sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2025-2026.bpmn

```diff
@@ -433,6 +433,7 @@ dmnX (comes from dmn)</bpmn:text>
           <zeebe:output source="=if studentDataDaycareCosts11YearsOrUnder = null&#10;then 0&#10;else&#10;  studentDataDaycareCosts11YearsOrUnder" target="studentDataDaycareCosts11YearsOrUnder" />
           <zeebe:output source="=if studentDataDaycareCosts12YearsOrOver = null&#10;then 0&#10;else&#10;  studentDataDaycareCosts12YearsOrOver" target="studentDataDaycareCosts12YearsOrOver" />
           <zeebe:output source="=if calculatedDataScholarshipsBursaries = null&#10;then 0&#10;else&#10;  calculatedDataScholarshipsBursaries" target="calculatedDataScholarshipsBursaries" />
+          <zeebe:output source="=if programYearTotalPartTimeBooksAndSuppliesCost = null&#10;then 0&#10;else&#10;programYearTotalPartTimeBooksAndSuppliesCost" target="programYearTotalPartTimeBooksAndSuppliesCost" />
```

Comment:
We have a section dedicated to other `programYearTotal` variables. Can we keep them all together?
It is now taking care of the awards, but it was also intended to take care of those.

![image](https://github.com/user-attachments/assets/31c435e5-8a08-4694-acfd-3a076cb0ccf0)